Assets/Scripts/Customization/CharacterCustomization.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[ExecuteInEditMode]
+public class CharacterCustomization : MonoBehaviour {
+    public static CharacterCustomization instance;

```

Comment:
Why is this singleton? I don't see it used anywhere

Reply:
same story as for the hair customization